hedera-node/src/test/java/com/hedera/services/store/models/TokenRelationshipTest.java

```diff
@@ -69,6 +72,10 @@ void toStringAsExpected() {
 		assertEquals(desired, subject.toString());
 	}
 
+	@Test
+	void balanceAdjustmen() {
```

Comment:
nit: typo. Should be `void balanceAdjustment() {`

Reply:
This test is removed in #1871